IE11 Support

MFalbo opened the pull request:
Changes for IE11 support, and strict dependency injection for minimization support

MFalbo committed:
IE11 and minimization support

Changes for IE11 support, and strict dependency injection for minimization support

MFalbo committed:
IE11 and minimization support

Changes for IE11 support, and strict dependency injection for minimization support

MFalbo committed:
IE11 and minimization support

Changes for IE11 support, and strict dependency injection for minimization support

ksheedlo commented on src/angular-select-text.js:
```diff
@@ -3,21 +3,21 @@
         range.selectNode(element[0]);
         $window.getSelection().addRange(range);
-      };
+     };
```
nit: indentation needs to be fixed.

ksheedlo commented on src/angular-select-text.js:
```diff
@@ -28,4 +28,4 @@ angular.module('angular-select-text', []).
     };
-  });
\ No newline at end of file
+  }]);
```
Did you chop off the end of the directive definition?

ksheedlo replied:
Never mind, it looks like there's a bug in the diff viewer.

eddywashere committed:
Merge pull request #8 from eddywashere/revert-6-revert-4-master

use strict dependency injection

ksheedlo commented:
@MFalbo we have addressed strict dependency injection in #4. Can you rebase against latest master and change this PR so that it is only addressing IE11?

Also, more explanation about what APIs need to be used to support IE11 would be super helpful!

ksheedlo commented on src/angular-select-text.js:
```diff
@@ -3,21 +3,21 @@
+  directive('selectText', ['$window', function ($window) {
     var selectElement;
 
-    if ($window.document.selection) {
```
Is `document.selection` actually a thing? I can't find anything about it in the Mozilla docs.

Commit:
update to 1.1.3

Commit:
Merge branch 'master' of https://github.com/MFalbo/angular-select-text

Commit:
IE11 Support

Commit:
IE11 Support

Commit:
Merge branch 'master' of https://github.com/MFalbo/angular-select-text

MFalbo commented:
Rebased and fixed indentation issue.  This code should work with current and older versions of IE as well as other modern broswers.  'Selection' lost support with IE11 (https://msdn.microsoft.com/library/ms535869(v=vs.85).aspx)
